Src/IronPython/Runtime/Operations/PythonOps.cs

```diff
@@ -939,7 +939,8 @@ public static partial class PythonOps {
                 return os.Length;
             }
 
-            object len = PythonContext.InvokeUnaryOperator(DefaultContext.Default, UnaryOperators.Length, o, "len() of unsized object");
+            object len = PythonContext.InvokeUnaryOperator(DefaultContext.Default, UnaryOperators.Length, o, 
```

Comment:
Would be nicer to add a `PythonContext.TryInvokeUnaryOperator` and only build the error string on failure.